libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1491,4 +1506,89 @@ public static String markMediaFailed(Context context, @NonNull String postConten
         }
         return postContent;
     }
+
+    public static boolean hasMediaItemsMarkedUploading(Context context, @NonNull String postContent) {
+        return hasMediaItemsMarkedWithTag(context, postContent, ATTR_STATUS_UPLOADING);
+    }
+
+    public static boolean hasMediaItemsMarkedFailed(Context context, @NonNull String postContent) {
+        return hasMediaItemsMarkedWithTag(context, postContent, ATTR_STATUS_FAILED);
+    }
+
+    private static boolean hasMediaItemsMarkedWithTag(Context context, @NonNull String postContent, String tag) {
+        // fill in Aztec with the post's content
+        AztecText content = new AztecText(context);
+        content.fromHtml(postContent);
+
+        // get all items with "uploading" class
```

Comment:
This comment appears incorrect - we're getting all the items with the passed class, not the "uploading" class.